src/main/java/com/loofah/graph/api/models/Skill.java

```diff
@@ -0,0 +1,39 @@
+package com.loofah.graph.api.models;
+
+public class Skill {
```

Comment:
Are we banning Lombok on our project 😃

Reply:
To play devil's advocate - what would be the main reason for not using it for loofah? On the face of it it seems like it would save a decent amount of time, especially if we want builders etc.

Reply:
I'm not super fussy but I would er towards the side of banning Lombok as I find it more transparent being able to see the methods 😬

Reply:
I'm easy either way, I'm going to leave it out for now, and will let whoever need to write builders make the call at that point 😛